research-seminar.md

```diff
@@ -6,24 +6,29 @@ order: 2
 
 <div class="message">
   The Chaincode Research Seminar exists to foster and encourage collaboration between the open-source community, industry, and academia towards research advances that benefit Bitcoin.
-</div>
```

Comment:
unneeded space

Reply:
Thanks. I did notice this when I viewed the diff on github. Next time I'll fix things like this.